src/services/getEvmBalances.ts

```diff
@@ -57,30 +57,33 @@ const getTokensBalanceSupportingMultiCall = async (
     tryAggregate: true
   });
 
-  const { results } = (await multicallInstance.call(contractCallContext)) ?? {
-    results: {}
-  };
+  try {
+    const { results } = (await multicallInstance.call(contractCallContext)) ?? {
+      results: {}
+    };
+    const tokenBalances: TokenBalance[] = [];
 
-  const tokenBalances: TokenBalance[] = [];
+    for (const symbol in results) {
+      const data = results[symbol].callsReturnContext[0] ?? {};
 
-  for (const symbol in results) {
-    const data = results[symbol].callsReturnContext[0] ?? {};
+      const { decimals = 18, address = "0x" } =
```

Comment:
why doing `address = 0x"`

Reply:
Was a fallback in case no token was found.
Just fixed it skipping iteration if no token is found